great-idea/index.html

```diff
@@ -15,9 +15,67 @@
 </head>
 
   <body>
-
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
-    
+  <header>
+  <nav class="navigation">
+    <a href="services.html">Services</a>
+    <a href="">Product</a>
+    <a href="">Vision</a>
+    <a href="">Features</a>
+    <a href="">About</a> 
+    <a href="">Contact</a> 
+  
+  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+  </nav>
+</header>
+  <div class="container">
+    <section class="head">
+  <h1 class="innovation">
+  Innovation<br>
+  On<br>
+  Demand
+  </h1>
+ 
+  <div class="button">Get Started</div>
+  <img src="img/header-img.png" alt="Image of a code snippet.">
+    </section>
+  <section> 
```

Comment:
Good use of the `<section>` tag here, making your code more semantic. You could add the class "top-content" to the section tag and just get rid of the `<div>` completely, as it's kind of unnecessary to have two containers here